homeassistant/components/unifiled/light.py

```diff
@@ -0,0 +1,106 @@
+"""Support for Unifi Led lights."""
+import logging
+
+import voluptuous as vol
+
+from unifiled import unifiled
+from homeassistant.components.light import (
+    ATTR_BRIGHTNESS,
+    PLATFORM_SCHEMA,
+    SUPPORT_BRIGHTNESS,
+    Light,
+)
+from homeassistant.const import CONF_HOST, CONF_PASSWORD, CONF_PORT, CONF_USERNAME
+import homeassistant.helpers.config_validation as cv
+
+_LOGGER = logging.getLogger(__name__)
+
+# Validation of the user's configuration
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
+    {
+        vol.Required(CONF_HOST): cv.string,
+        vol.Required(CONF_USERNAME): cv.string,
+        vol.Required(CONF_PASSWORD): cv.string,
+        vol.Optional(CONF_PORT, default="20443"): cv.string,
+    }
+)
+
+
+def setup_platform(hass, config, add_entities, discovery_info=None):
+    """Set up the Unifi LED platform."""
+
+    # Assign configuration variables.
+    # The configuration check takes care they are present.
+    _ip = config[CONF_HOST]
```

Comment:
Please don't prefix local variables with underscore. It's not needed.

Reply:
Changed this